main.go

```diff
@@ -118,20 +120,41 @@ func main() {
 		if *test {
 			logrus.Info("crontab is valid")
 			os.Exit(0)
-			break
 		}
 
 		var wg sync.WaitGroup
 		exitCtx, notifyExit := context.WithCancel(context.Background())
 
-		for _, job := range tab.Jobs {
-			cronLogger := logrus.WithFields(logrus.Fields{
-				"job.schedule": job.Schedule,
-				"job.command":  job.Command,
-				"job.position": job.Position,
-			})
+		if firstRun {
```

Comment:
I'd rather keep the code a little simpler and re-run the start jobs if you reload the crontab.

There are 2 reasons for this:

- First, it keeps the code simpler. We don't need 2 blocks to run start jobs and regular jobs separately, etc.
- It enables the user to do more. With the approach you have here, you cannot run `@start` jobs when reloading the crontab. With the approach I'm proposing, you can choose to not run them (remove them from the crontab before reloading), or to run them (don't remove them).

`@start` might be a bit un-ideal as a name then. In the issue folks were mentioning using `@reboot`. I think that'd work well here.

Reply:
Yea, I think restarting the cron jobs when container restarts is very reasonable.

Reply:
i think it is not a good idea to run all `@start` jobs again on a reload. if i decleare a job with `@start` i want the job to run only once at container start.
if we want to be compatible with other cron scheduler we should use `@reboot`.
if the job should be executed every time the crontab is reloaded, `@reboot` is a bad name option. mabye `@reload` is a better option to make it clear, but I'm not happy with it either.

Reply:
Unlike the other review comments, it seems this needs a decision, on two points:

* Name. I agree that compatibility with `@reboot` sounds like a good idea, rather than a custom `@start`.
* Whether to run on reload or not. For compatibility with other cron systems, I agree it should not run on reload. This code could probably be simplified. There's no "on load" functionality in [Vixie's `crontab`](https://www.man7.org/linux/man-pages/man5/crontab.5.html), so I don't understand why the flexibility would be worth having (without a specific usecase.) That said, I store the crontab in container images, so I never use reload.